testnet/stacks-node/src/config.rs

```diff
@@ -445,9 +506,18 @@ impl Config {
                             // Using std::net::LookupHost would be preferable, but it's
                             // unfortunately unstable at this point.
                             // https://doc.rust-lang.org/1.6.0/std/net/struct.LookupHost.html
-                            let mut addrs_iter =
-                                format!("{}:1", peer_host).to_socket_addrs().unwrap();
-                            let sock_addr = addrs_iter.next().unwrap();
+                            let mut sock_addrs = format!("{}::1", &peer_host)
+                                .to_socket_addrs()
+                                .map_err(|e| format!("Invalid [burnchain]peer_host: {}", &e))?;
```

Comment:
On line 608 we are using the notation `burnchain.local_mining_public_key` but here you are using `[burnchain]peer_host`.

I think the first one makes more sense to use consistently.

Reply:
Great feedback! I updated to use preexisting naming convention.